src/NUnitEngine/nunit.engine/Services/ServiceManager.cs

```diff
@@ -53,7 +54,11 @@ public IService GetService( Type serviceType )
             else
                 foreach( IService service in _services )
                 {
+#if NETSTANDARD1_3
+                    if (serviceType.GetTypeInfo().IsAssignableFrom(service.GetType().GetTypeInfo()))
+#else
                     if( serviceType.IsInstanceOfType( service ) )
+#endif
```

Comment:
This would enable unifying the code:

https://github.com/nunit/nunit/blob/6af1a1536e2ad86d3974b1167cb3c984b0c23f83/src/NUnitFramework/framework/nunit.framework.csproj#L22-L24